OpenRA.Mods.Common/Traits/HitShape.cs

```diff
@@ -74,13 +79,19 @@ protected override void Created(Actor self)
 		{
 			orientation = self.Trait<BodyOrientation>();
 			targetableCells = self.TraitOrDefault<ITargetableCells>();
+			turret = self.TraitsImplementing<Turreted>().SingleOrDefault(t => t.Name == Info.LinkedTurret);
```

Comment:
We use `FirstOrDefault` (or just `First`) everywhere else, so best to be consistent here.